future/src/ct/ct_storage_sqlite.h

```diff
@@ -57,6 +57,20 @@ class CtStorageSqlite : public CtStorageEntity
     Gtk::TreeIter       _node_from_db(guint node_id, Gtk::TreeIter parent_iter);
     CtTreeIter          _node_from_imported_db(gint64 node_id, CtTreeIter* parent_iter);
     
+    /**
+     * @brief Check that the database contains the required tables
+     * 
+     * @param db 
+     */
+    void                _check_db_tables(sqlite3* db);
```

Comment:
wrong signature

Reply:
in .cc its _check_db_tables()

Reply:
Yep, I pushed it accidentally :facepalm: (meant to do `git diff`, history was `git push`)